ui/src/kapacitor/components/config/SMTPConfig.js

```diff
@@ -8,27 +8,29 @@ class SMTPConfig extends Component {
     }
   }
 
-  handleSubmit = e => {
+  handleSubmit = async e => {
     e.preventDefault()
 
     const properties = {
       host: this.host.value,
       port: this.port.value,
       from: this.from.value,
+      to: this.to.value ? [this.to.value] : [],
```

Comment:
Are we supporting just one destination email?

Reply:
in the config, yes. You can add more in the alert rule.

Reply:
@ebb-tide let's keep this as a single email for now and we can tackle that in https://github.com/influxdata/chronograf/issues/2778